.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+<!--
+Set the PR title to a meaningful commit message in imperative form. E.g.:
+
+clp-s: Don't add implicit wildcards ('*') at the beginning and the end of a query (fixes #390).
```

Comment:
This message might not make too much sense for stand-alone developers in this project